conda_forge_tick/auto_tick.xsh

```diff
@@ -355,6 +355,56 @@ def add_rebuild_successors(migrators, gx, package_name, pin_version, pr_limit=5,
                 top_level=top_level,
                 cycles=cycles, obj_version=obj_version))
 
+def add_rebuild_rlang(migrators, gx, package_name, pin_version, pr_limit=10, obj_version=0):
```

Comment:
Can you refactor to avoid code duplication?

I would change the signature of `add_rebuild_successors` to
`
def add_rebuild_successors(migrators, gx, package_name, pin_version, pr_limit=5, obj_version=0, rebuild_class=Rebuild):
`

and then change,
`add_rebuild_rlang($MIGRATORS, gx, 'r-base', '3.6.1')`
to
`add_rebuild_successors($MIGRATORS, gx, 'r-base', '3.6.1', rebuild_class=RBaseRebuild)`